src/OpenTelemetry.Exporter.Geneva/Common.GenevaExporter.props

```diff
@@ -1,9 +1,5 @@
 <Project>
-  <PropertyGroup>
-    <OTelSdkVersion>$(OpenTelemetryCoreLatestVersion)</OTelSdkVersion>
```

Comment:
The purpose of `OTelSdkVersion` is so that GenevaExporter can toggle between `OpenTelemetryCoreLatestVersion` and `OpenTelemetryCoreLatestPrereleaseVersion` and optionally depend on experimental features when in prerelease mode.

We may not use it again in which case we can just remove this entire `.props` file but I think for now let's leave it alone.

Reply:
> The purpose of OTelSdkVersion is so that GenevaExporter can toggle between OpenTelemetryCoreLatestVersion and OpenTelemetryCoreLatestPrereleaseVersion

Is it the same for OneCollector?

Reply:
Ya. See this block: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/508405465a83af8d625b22c6559929c79c2ec0a3/src/OpenTelemetry.Exporter.OneCollector/Internal/Serialization/LogRecordCommonSchemaJsonSerializer.cs#L138

`EXPOSE_EXPERIMENTAL_FEATURES` should be set for prerelease builds. `OpenTelemetryCoreLatestVersion` should always be a stable version. So these `OTelSdkVersion` properties are only around to easily toggle to prereleases without having to update a bunch of places in the code.